ciongke/cmd/hook/main.go

```diff
@@ -61,7 +61,7 @@ var defaultJenkinsJobs = map[string][]JenkinsJob{
 		{
 			Name:      "kubernetes-pull-build-test-kubemark-e2e-gce",
 			Trigger:   regexp.MustCompile(`@k8s-bot kubemark (e2e )?test this`),
-			AlwaysRun: false,
+			AlwaysRun: true,
```

Comment:
I'm assuming that for now, those tests still won't be blocking right?
[Ideally I would like them to be blocking at some point, but we need to verify that it really works first.]